### ISSUE TITLE
[python] bucket parameter error with autotune 

### ISSUE THREAD

**Issue by linetor** [facebookresearch/fastText#1017]
I trained with autotune, and I found that with pre-defined bucker, there is always Floating point exception (core dumped).

with verbose=4, it's bucket's number is always 0.

And I check with command line, it show normal output.

python's output

>>> classifier = fasttext.train_supervised(
...             input="/pang/part1-translate_checking_with_only_productname_with_sf.traing"
...     ,autotuneValidationFile="/pang/part2-translate_checking_with_only_productname_with_sf.traing"
...             ,autotuneDuration=60*60*1
...             ,verbose=4
...     ,bucket=2000000
...         )
Warning : bucket is manually set to a specific value. It will not be automatically optimized.
Trial = 1
epoch = 5
lr = 0.1
dim = 100
minCount = 1
wordNgrams = 1
minn = 0
maxn = 0
bucket = 0
dsub = 2
loss = softmax
Progress:   0.1% Trials:    1 Best score:   unknown ETA:   0h59m54scurrentScore = 0.997203
train took = 5.24087`
Trial = 2
epoch = 2
lr = 0.169298
dim = 105
minCount = 1
wordNgrams = 5
minn = 3
maxn = 6
bucket = 0
dsub = 2
loss = softmax
Progress:   0.2% Trials:    2 Best score:  0.997203 ETA:   0h59m53sFloating point exception (core dumped)
